climbing things does not stop tail wagging

Sniblet opened the pull request:
here we go again

# Document the changes in your pull request

Tested, works:
Climbing structures (tables) and machines (???) does not stop tail wagging.

Someday soon:
[Slipping does not stop tail wagging](https://github.com/yogstation13/Yogstation/pull/13452). Left out of this PR because `I have no idea how slipping works` I mean I'm atomizing.

# Changelog

:cl: 
bugfix: climbing things doesn't discourage waggers
tweak: felinids can now oppress other species
/:cl:

Sniblet committed:
Update _machinery.dm

tattax reviewed (changes requested):
define H by using var/mob/living/carbon/human/H

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update _machinery.dm

i ask myself why im the one doing this and then i remember its mostly out of spite, apparently everyone else can do this easily but im the only one who cares about the result

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Merge branch 'yogstation13:master' into still-not-over

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update structures.dm

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update structures.dm

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update structures.dm

blindlet

Sniblet committed:
Update structures.dm

brainlet

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update structures.dm

Sniblet committed:
Update slippery.dm

wait it already has forcedrop? whats the stun even for then lol??

Sniblet committed:
Update atoms.dm

Sniblet committed:
Update open.dm

Sniblet committed:
Update carbon_movement.dm

Sniblet committed:
Update human_movement.dm

Sniblet committed:
Update mob_movement.dm

Sniblet committed:
Update structures.dm

Sniblet committed:
Update _machinery.dm

Sniblet committed:
Update slippery.dm

Sniblet committed:
Update atoms.dm

Sniblet committed:
Update open.dm

Sniblet committed:
Update open.dm

Sniblet committed:
Update carbon_movement.dm

Sniblet committed:
Update human_movement.dm

Sniblet committed:
Update mob_movement.dm

ToasterBiome reviewed (dismissed):
this is a weird way to do it, also you better check if they are indeed a living/carbon/human for safety™

Sniblet committed:
Update code/game/machinery/_machinery.dm

Co-authored-by: Byemoh <baiomurang@gmail.com>

Sniblet committed:
Update code/game/objects/structures.dm

Co-authored-by: Byemoh <baiomurang@gmail.com>
